VERSION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
Btw, I am not sure why this file is here and what it does, do you have more context @microbit-sam ?